public/v4/apps/forge_minecraft.yml

```diff
@@ -0,0 +1,98 @@
+captainVersion: 4
+services:
+  $$cap_appname:
+    image: itzg/minecraft-server:latest
```

Comment:
Avoid using `latest` tag due to the fact that it's mutable. Use a specific version, like `2023.1.0-java17`

Reply:
I added `java19` as the current latest version